app/views/LocationTracking.js

```diff
@@ -304,6 +305,20 @@ class LocationTracking extends Component {
     );
   }
 
+  //Change pulse location on a few problematic devices.
+  getPulseStyle() {
```

Comment:
We haven't done a great job of documenting our [style guides](https://github.com/tripleblindmarket/covid-safe-paths/wiki/Code-styleguide#use-switch-statements), but one thing we've been trying to keep consistent is the usage of switch statements for blocks of logic with more than one or two `else if`s. Could you refactor to something more like this?

```js
getPulseStyle() {
  switch(height) {
    case height > 900:
      return { top: '29%' };
  }
}